package.json

```diff
@@ -40,11 +40,7 @@
     "release": "npm run build && changeset publish"
   },
   "devDependencies": {
-    "@types/jest": "^27.0.2",
```

Comment:
There was one point in time where our Jest types were not working from the package level, so we added `@types/jest` to the monorepo root here. It looks like this isn't the case anymore though. If it somehow crops up in the future we can just add this back.